jujugui/static/gui/src/app/components/env-list/env-list.js

```diff
@@ -101,7 +103,10 @@ YUI.add('env-list', function() {
         <juju.components.Panel
           instanceName="env-list-panel"
           visible={true}>
-          <ul className="env-list">
+          <ul className="env-list"
+            role="menu"
+            aria-hidden="true"
```

Comment:
As discussed, i will remove the aria-hidden attribute as it is only visible as true. Therefore useless.